main.py

```diff
@@ -1,5 +1,22 @@
 def runPersonDatabase():
-	pass
+	
+	for i in range(3):
+		userInput = inputUser()
+		personDict = createPerson(*userInput)
+		print(personDict)
+
+def inputUser():
+	fname = input("What is the users first name: ")
```

Comment:
I think its better to write full variable name to understand better

Reply:
I agree and I corrected it.